corehq/apps/cloudcare/static/cloudcare/js/form_entry/webformsession.js

```diff
@@ -495,7 +503,9 @@ hqDefine("cloudcare/js/form_entry/webformsession", function () {
         };
 
         self.serverError = function (q, resp) {
-            if (resp.type === "required") {
+            if (!resp) {
+                q.serverError(null);
+            } else if (resp.type === "required") {
```

Comment:
While you're in here, could you add translations for the user-facing errors? They just need to be wrapped in `gettext` calls.